package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "isoly",
-	"version": "2.0.25",
+	"version": "2.1.0",
```

Comment:
Why do you bump minor here? First of all why is a minor bump needed and second doing it like this together with the feat: prefix in your commit would create a double minor bump.

Reply:
I don't think the utily repos have, rules to bump on `feat: ` and `fix: ` etc.
In utily everything gets a patch-bump with each PR, so this will be version `2.1.1` after this PR is merged.

Reply:
It does, Github actions is using "phips28/gh-action-bump-version@master" this tool works with commit message keywords.

Reply:
It should hopefully not change anything from an outside perspective, it works when importing it in langly, so I assume it would only have an effect on direct imports of distributed files, which is not often done in utily (or Intergiro acquiring code) as far as I know. 
But in case it would change it for anyone I felt like a feature/minor bump would make sense.
As Magnus said I assumed utily repositories don't do minor bumps on their own, so I made one myself, assuming it would turn into 2.1.1, which I felt was acceptable

Reply:
Changed it to build now, missed your comment earlier